Signal/src/Models/TSMessageAdapaters/TSMessageAdapter.m

```diff
@@ -217,7 +217,8 @@ - (NSString *)senderId {
 }
 
 - (NSString *)senderDisplayName {
-    if (self.thread) {
+    BOOL isSenderMe = [_senderDisplayName isEqualToString:NSLocalizedString(@"ME_STRING", @"")];
```

Comment:
Certainly. Wasn't sure which between the two would be more robust.

Should I push --force the changed commits to this branch?